x-pack/plugins/observability/public/hooks/use_alerts_history.ts

```diff
@@ -30,14 +31,19 @@ interface Props {
   };
 }
 interface FetchAlertsHistory {
-  totalTriggeredAlerts: number;
-  histogramTriggeredAlerts: estypes.AggregationsDateHistogramBucketKeys[];
-  error?: string;
-  avgTimeToRecoverUS: number;
+  totalTriggeredAlerts?: number;
+  histogramTriggeredAlerts?: estypes.AggregationsDateHistogramBucketKeys[];
+  avgTimeToRecoverUS?: number;
+}
+
+export interface UseAlertsHistory extends FetchAlertsHistory {
+  loading: boolean;
+  error?: Error;
+  refetch: () => Promise<FetchAlertsHistory>;
 }
 
-export function useAlertsHistory({ featureIds, ruleId, dateRange }: Props) {
-  const { http } = useKibana<InfraClientCoreStart>().services;
+export function useAlertsHistory({ featureIds, ruleId, dateRange }: Props): UseAlertsHistory {
+  const { http } = useKibana<ObservabilityAppServices>().services;
```

Comment:
nit: I don't think we need to specify `ObservabilityAppServices` since we are importing `useKibana` from `../utils/kibana_react`

Reply:
done